### PR TITLE
Update text for #144

### PR DESCRIPTION
**Pull request by soulBit**
Please ask Spanish/Brazilian Portuguese translators to confirm the text I added for those languages!

### CODE REVIEW

**Comment by creed-victor** [comment]
@bitcoinuser @jjmr007 can you check for translation and suggest edits if needed?

@soulBit in case you are using google translator you can just use english translations in each of translation files - it should be more easily spottable for translators.
Also, only english language file keys required but filling other files should help translators a lot.

**Comment by bitcoinuser** [comment]
Portuguese Brazilian is ok.

**Comment by bitcoinuser** [comment]
Ativo e valor a ser colaterizado.

@soulBit please change to this.

**Comment by soulBit** [comment]
> @bitcoinuser @jjmr007 can you check for translation and suggest edits if needed?
> 
> @soulBit in case you are using google translator you can just use english translations in each of translation files - it should be more easily spottable for translators.
> Also, only english language file keys required but filling other files should help translators a lot.

Thanks, I actually got my flatmate to translate (but they are Spanish and non-crypto friendly). I've updated the pt-br text to match suggestion from @bitcoinuser

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -270,8 +270,7 @@
     },
     "borrowingContainer": {
       "amount": "Valor do empréstimo",
-      "token": "Ativo a ser colaterizado",
-      "collateralAmount": "Quantidade colaterizada",
+      "tokenAssetCollateral": "Ativo e valor a ser colaterizado",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:273]:
```suggestion
      "tokenAssetCollateral": "Ativo e quantidade a ser colaterizada",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
@soulBit please change again.

**Pull request merged**